File: frontend/src/Models/Application.ts
export type Application = {
  id: string;
  name: string;
  surname: string;
  jobOfferTitle: string;
  date: string;
  status: string;
  score: number | null;
  evaluation: string | null;
  cvId: number;
};

export type ApplicationGr = {
  id: string;
  name: string;
  surname: string;
  date: string;
  cvId: number;
  cvFileName: string;
  status: string;
  score: number | null;
  evaluation: string | null;
};

export type GroupedApplications = {
  jobOfferTitle: string;
  applications: ApplicationGr[];
};

File: api/Models/Application.cs
namespace api.Models
{
	public class Application
	{
		public Guid Id { get; set; }
		public DateTime Date { get; set; }
		public string Status { get; set; } = string.Empty;
		public int? Score { get; set; }
		public string? Evaluation { get; set; }


		public string AppUserId { get; set; } = string.Empty;
		public AppUser AppUser { get; set; } 
		public int JobOfferId { get; set; }
		public JobOffer JobOffer { get; set; } 
		public int CvId { get; set; }
		public CV CV { get; set; }
	}
}
